Extension/src/nativeStrings.json

```diff
@@ -443,5 +443,12 @@
     "unknown_error": "Unknown error.",
     "run_select_intellisense_configuration": {
         "text": "Please run the 'Select IntelliSense Configuration...' command to locate your system headers."
+    },"offer_copy_declaration": {
+        "text": "Copy declaration of '{0}' to clipboard",
```

Comment:
As mentioned elsewhere, I think the `to clipboard` should be `to the clipboard`.